src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -0,0 +1,1204 @@
+/*
```

Comment:
I haven't taken a look at it, but it is extremely easy. Only those functions that actually *do* something with mongo should stay in mongoBackend.